notes_for_class/source/index.rst

```diff
@@ -1,27 +0,0 @@
-
```

Comment:
don't delete anything outside your own students DIR!

you really need to read the git commit messages to make sure that you really want to do what you are doing.